pipe_segment/pipeline.py

```diff
@@ -185,11 +196,9 @@ def pipeline(self):
 
             if self.options.out_sat_offsets_table:
                 logger.info("Adding SatelliteOffsetsWrite transform...")
-
-                self.satellite_offsets_writer = SatelliteOffsetsWrite(
-                    self.options, self.cloud_options)
-                (
-                    satellite_offsets | self.satellite_offsets_writer
+                _ = (
```

Comment:
I mean we don't use it, but why it's assigned to a throwaway variable?

Reply:
unassigned expressions raise a warning on pyright. Not a huge thing, but I'm trying to fix that warning wherever I come across one of these.